### PR TITLE
Update padding, other style improvements

### PR DESCRIPTION
**Pull request by cameronblandford**
* Make padding consistent across sections
* Remove non-functional anchors
* Remove unnecessary information from item cards
* Change styling of contributor list, add links to users

### CODE REVIEW

File: app/components/collection-item/template.hbs [hunk 1]
```diff
@@ -17,7 +17,6 @@
                 <h4>{{#link-to 'collection.item' item.id}}{{item.title}}{{/link-to}}</h4>
             {{/if}}
             <p> Author: {{item.createdBy.computedFullName}} </p>
-            <p> Type: {{item.type}} </p>
```
Comment (cameronblandford) [inline app/components/collection-item/template.hbs]: As Jeff mentioned

File: app/components/contributor-list/template.hbs [hunk 2]
```diff
@@ -1,44 +1,41 @@
-<div id="{{get-section-id layout.title}}" class="meeting-keynote-speakers-container p-xl"
+<div id="{{get-section-id layout.title}}" class="meeting-keynote-speakers-container p-t-xl p-b-xl"
      style="{{containerStyle}}">
     <div class="container">
         {{#if layout.title}}
             <div class="row text-center section-title">
-                <h2 style="display: inline">
```
Comment (cameronblandford) [inline app/components/contributor-list/template.hbs]: Removed anchor link

File: app/components/contributor-list/template.hbs [hunk 3]
```diff
@@ -1,44 +1,41 @@
...
+                <h2>
                     {{layout.title}}
-                    <a class="headerlink" href="#{{get-section-id layout.title}}">
-                        <i class="fa fa-link"></i>
-                    </a>
                 </h2>
             </div>
         {{/if}}
-        <div class="row">
-            <div class="row">
```
Comment (cameronblandford) [inline app/components/contributor-list/template.hbs]: Remove extra row div to fix margin issue

File: app/styles/app.scss [hunk 4]
```diff
@@ -173,11 +178,20 @@ ul.comma-list {
     border-bottom: 1px solid #ddd;
 }
 
+#myNavbar {
+    padding: 0;
+}
+
+a.dropdown-toggle.nav-user-dropdown {
```
Comment (cameronblandford) [inline app/styles/app.scss]: Fix OSF nav padding issues

File: app/styles/components/meetings-schedule.scss [hunk 5]
```diff
@@ -2,7 +2,7 @@
   #schedule-window {
     height: 500px;
     overflow: auto;
-    padding: 15px;
+    padding: 15px 15px 15px 2px;
```
Comment (cameronblandford) [inline app/styles/components/meetings-schedule.scss]: Improve padding on schedule component

File: app/styles/components/meetings-schedule.scss [hunk 6]
```diff
@@ -37,7 +37,13 @@
   }
 
   .item-detail {
-    padding: 5px 30px 5px 5px;
+    padding: 0 0 5px 5px;
```
Comment (cameronblandford) [inline app/styles/components/meetings-schedule.scss]: Improve alignment on item detail of schedule component

File: app/components/landing-paragraph/template.hbs [hunk 7]
```diff
@@ -1,17 +1,20 @@
-<div id="{{get-section-id layout.title}}" class="landing-paragraph-container p-xl" style="{{containerStyle}}">
+<style>
```
Comment (birdbrained) [inline app/components/landing-paragraph/template.hbs]: We should put this in the css file

Comment (cameronblandford) [reply app/components/landing-paragraph/template.hbs]: Oh whoops, completely agreed. That's not supposed to be there.

**Pull request merged**